www/src/components/shell/onboarding/sections/cloud/provider/AWS.tsx

```diff
@@ -0,0 +1,94 @@
+import { useContext, useEffect, useMemo } from 'react'
+import { Flex } from 'honorable'
+import IsEmpty from 'lodash/isEmpty'
+
+import {
+  FormField,
+  Input,
+  ListBoxItem,
+  Select,
+} from '@pluralsh/design-system'
+
+import { OnboardingContext } from '../../../context/onboarding'
+import { IsObjectEmpty } from '../../../../../../utils/object'
+import { AWSCloudProvider, CloudProvider } from '../../../context/types'
+import { useSetCloudProviderKeys, useSetWorkspaceKeys } from '../../../context/hooks'
+
+const REGIONS = [
+  'af-south-1',
+  'eu-north-1',
+  'ap-south-1',
+  'eu-west-3',
+  'eu-west-2',
+  'eu-south-1',
+  'eu-west-1',
+  'ap-northeast-3',
+  'ap-northeast-2',
+  'me-south-1',
+  'ap-northeast-1',
+  'sa-east-1',
+  'ca-central-1',
+  'ap-east-1',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'eu-central-1',
+  'ap-southeast-3',
+  'us-east-1',
+  'us-east-2',
+  'us-west-1',
+  'us-west-2',
```

Comment:
It would be nice to use some library or API for that in the future.

Reply:
Ye it would be nice to have it for all providers and move to the API. This list was already there. I think it only contains regions that actually work with plural. It is definitely not a full list.